app/lib/actions/katello/content_view_puppet_module/destroy.rb

```diff
@@ -10,8 +10,9 @@ def plan(repository)
           # puppet modules that are referring to it
           repository.puppet_modules.each do |puppet_module|
             # first, process content view puppet modules that have been specified by version
-            library_repos = ::Katello::Repository.in_environment(repository.organization.library).
-                                                  where(:pulp_id => puppet_module.repoids)
+            library_repos = ::Katello::Repository.
+              in_environment(repository.organization.library).
+              where(:pulp_id => puppet_module.repository_ids)
```

Comment:
Not sure if I see the need to move this from 2 to 3 lines unless its throwing rubocop errors